### PR TITLE
Addressed QA feedback

### PR DESCRIPTION
**Pull request by AnumehaSrivastava05**
Added help tooltip
Corrected date format
Added security

### CODE REVIEW

File: frontend/src/views/patientregistration/ViewPatientRegistration.vue [hunk 1]
```diff
@@ -176,7 +204,6 @@ export default {
     },
     resetForm() {
       this.phn = ''
-      this.payee = ''
```
Comment (daveb-hni) [inline frontend/src/views/patientregistration/ViewPatientRegistration.vue:179]: Why is this removed, payee field should be reset also?

**Review by daveb-hni — CHANGES_REQUESTED** [review]
Please see comments

File: frontend/src/views/patientregistration/ViewPatientRegistration.vue (hunk 1 above)
Comment (AnumehaSrivastava05) [reply frontend/src/views/patientregistration/ViewPatientRegistration.vue:179]: It will be a read only field once https://dev.azure.com/cgi-vic-hlth/HNWeb/_workitems/edit/7102 is done.

**Review by daveb-hni — APPROVED** [review]
Changes look good
